open-media-match/src/OpenMediaMatch/tests/test_api.py

```diff
@@ -0,0 +1,23 @@
+import os
+import pytest
+
+from OpenMediaMatch import create_app
+
+
+@pytest.fixture()
+def app():
+    os.environ.setdefault("OMM_CONFIG", "tests/omm_config.py")
+    app = create_app()
+
+    yield app
+
+
+@pytest.fixture()
+def client(app):
+    return app.test_client()
+
+
+def test_status_response(client):
```

Comment:
Ooo, nice, e2e tests

Reply:
We can definitively have both e2e and unit ( which I prefer. We can also split the test jobs if it becomes slow), but given the current state e2e sounded the way to go for a good sample for now.